code/game/objects/items/weapons/medigun.dm

```diff
@@ -0,0 +1,360 @@
+/////Medi-Gun, coded by Deity Link, inspired by Valve's Team Fortress 2/////
```

Comment:
It'll certainly help the coders of the future if they ever have to go "WHO CODED THIS PIECE OF SHIT"